hrrecruitment/tests/phpunit/CRM/HRRecruitment/BAO/HRSearchVacancyTest.php

```diff
@@ -1,37 +1,29 @@
 <?php
-/*
- +--------------------------------------------------------------------+
- | CiviHR version 1.4                                                 |
- +--------------------------------------------------------------------+
- | Copyright CiviCRM LLC (c) 2004-2014                                |
- +--------------------------------------------------------------------+
- | This file is a part of CiviCRM.                                    |
- |                                                                    |
- | CiviCRM is free software; you can copy, modify, and distribute it  |
- | under the terms of the GNU Affero General Public License           |
- | Version 3, 19 November 2007 and the CiviCRM Licensing Exception.   |
- |                                                                    |
- | CiviCRM is distributed in the hope that it will be useful, but     |
- | WITHOUT ANY WARRANTY; without even the implied warranty of         |
- | MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.               |
- | See the GNU Affero General Public License for more details.        |
- |                                                                    |
- | You should have received a copy of the GNU Affero General Public   |
- | License along with this program; if not, contact CiviCRM LLC       |
- | at info[AT]civicrm[DOT]org. If you have questions about the        |
- | GNU Affero General Public License or the licensing of CiviCRM,     |
- | see the CiviCRM license FAQ at http://civicrm.org/licensing        |
- +--------------------------------------------------------------------+
-*/
-
-require_once 'CiviTest/CiviUnitTestCase.php';
-require_once 'CiviTest/Contact.php';
-class CRM_HRRecruitment_BAO_HRSearchVacancyTest extends CiviUnitTestCase {
+
+use Civi\Test\HeadlessInterface;
+use Civi\Test\TransactionalInterface;
+
+/**
+ * Class CRM_HRRecruitment_BAO_HRSearchVacancyTest
+ *
+ * @group headless
+ */
+class CRM_HRRecruitment_BAO_HRSearchVacancyTest extends CiviUnitTestCase implements HeadlessInterface , TransactionalInterface {
```

Comment:
I don't see this class using anything from CiviUnitTestCase. You can just extend PHPUnit_Framework_TestCase here